src/v/cluster/self_test_rpc_types.h

```diff
@@ -341,7 +341,7 @@ struct empty_request
 struct start_test_request
   : serde::envelope<
       start_test_request,
-      serde::version<1>,
```

Comment:
ISTR we maybe _don't_ need to bump if the difference in version doesn't span a release? Were cloudcheck_opts and unknown_checks added in the same version? Or was one backported but not the other?

Reply:
`unknown_checks` exists as of `v24.1.x` due to backports, `cloudcheck_opts` is only in `v24.2.x`.